### PR TITLE
Adding V&V case compos. dep. model to test cases.

### PR DESCRIPTION
**Pull request by sjhbtue**
## Proposed Changes
Re-adding required files for test case VV sandia jet and changed regression.yml to get residual values required for vandv.py
@bigfooted @Cristopher-Morales. (This PR is created to get the residual values for the vandv.py script.)
 

## PR Checklist

- [X] I am submitting my contribution to the develop branch.
- [X] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [X] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [X] I have added a test case that demonstrates my contribution, if necessary.
- [ ] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

**Comment by sjhbtue** [comment]
@pcarruscag In this PR, I have a problem with running the regression tests, because it can not find my branch, as can be seen in the figure below:
image
I believe it might be related to the fact that I'm creating a PR from a created fork of the SU2 repository, instead of just having an own branch in the su2code/SU2 repository. So, the branch I'm in is located in sjhbtue/SU2. Could this indeed be the problem or do you think I can still solve it in a different way? Thanks in advance!

File: .github/workflows/regression.yml [hunk 1]
```diff
@@ -171,7 +171,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:230704-1323
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c feature_validation_sandia_jet -s ${{matrix.testscript}}
```
Comment (pcarruscag) [inline .github/workflows/regression.yml:174]:
```suggestion
          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
```

File: TestCases/vandv.py [hunk 2]
```diff
@@ -85,6 +85,18 @@ def main():
...
+    ##########################
+    ### Incompressible RANS ###
+    ##########################
+
+    # Sandia jet - sst-v2003m
+    sandiajet_sst           = TestCase('sandiajet_sst')
+    sandiajet_sst.cfg_dir   = "vandv/species_transport/sandia_jet"
+    sandiajet_sst.cfg_file  = "VALIDATION.cfg"
+    sandiajet_sst.test_iter = 5
+    sandiajet_sst.test_vals = [-11.527743, -11.150388, -11.944923, -10.750834, -11.116769, -4.030059]
```
Comment (pcarruscag) [inline TestCases/vandv.py:97]:
```suggestion
    sandiajet_sst.test_vals = [-16.249917, -13.835991, -14.303372, -13.276035, -10.074262, -14.027223, 5, -1.672359, 5, -4.938477, 5, -3.462217, 2.5859e-04, 2.8215e-32, 4.5010e-68, 2.5859e-04, 4.0474e+03, 3.9468e+03, 4.9170e+01, 5.1441e+01]
```

File: TestCases/vandv/species_transport/sandia_jet/VALIDATION.cfg [hunk 3]
```diff
@@ -0,0 +1,151 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Comment (pcarruscag) [inline TestCases/vandv/species_transport/sandia_jet/VALIDATION.cfg:1]: please rename this file to lower case

**Comment by sjhbtue** [comment]
Thank you! @pcarruscag @Cristopher-Morales

**Pull request merged**